app-lifelike/src/main/java/com/example/lifelike/composable/loggedout/Splash.kt

```diff
@@ -21,20 +19,20 @@ interface Splash {
 
         @Composable
         fun Content(onNext: () -> Unit) {
-            Column(modifier = LayoutPadding(40.dp), arrangement = Arrangement.SpaceAround) {
+            Column(modifier = Modifier.padding(40.dp), arrangement = Arrangement.SpaceAround) {
                 Text(
                     text = AnnotatedString(
                         text = "Welcome to amazing fake app",
                         paragraphStyle = ParagraphStyle(
                             textAlign = TextAlign.Center
                         )
                     ),
-                    style = MaterialTheme.typography().h4
+                    style = MaterialTheme.typography.h4
                 )
-                Container(
-                    expanded = true,
-                    constraints = DpConstraints(maxHeight = 48.dp),
-                    alignment = Alignment.Center
+                Box(
+                    modifier = Modifier.fillMaxSize()
+                            + Modifier.preferredSizeIn(DpConstraints(maxHeight = 48.dp))
+                            + Modifier.wrapContentSize(Alignment.Center)
```

Comment:
Let's use fluent api for modifiers here
```suggestion
                    modifier = Modifier
                            .fillMaxSize()
                            .preferredSizeIn(DpConstraints(maxHeight = 48.dp))
                            .wrapContentSize(Alignment.Center)
```